ctsit/move_data_to_other_event
# Fix no username crash

Addresses a bug reported by an admin where they were getting lots of module crashes. The crash was caused by someone attempting to access the Record Status Dashboard page while not logged in. Not really fatal since the module would have exited anyway since the non-user can't see the page anyway, but it's annoying admins.

To test:
1. Create a project and activate the module
1. Open a private browsing window and copy the URL of any project page with the module active
1. Watch mailhog for a message with "Exception: A username was not specified and could not be automatically detected."
1. Activate this PR
1. Reload the private browsing page
1. Observe the message is no longer sent

Also adds a test project XML, preloaded with a pdf.
